Fix linting issues

Manual linting using ESLint has not worked on some machines. 

Under scripts there is ` "lintjs": "npx eslint '**/*.js'"`, the `npx eslint '**/*.js' command works as expected but `npm run lintjs`  breaks.

```
→ D:\GitHub\CSE110 Project\cse110-fa21-group2 [week7-githubactions ≡ +0 ~2 -0 !]› npm run lintjs
> cse110-fa21-group2@1.0.0 lintjs D:\GitHub\CSE110 Project\cse110-fa21-group2
> npx eslint '**/*.js'
Oops! Something went wrong! :(
ESLint: 8.2.0
No files matching the pattern "'**/*.js'" were found.
Please check for typing mistakes in the pattern.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! cse110-fa21-group2@1.0.0 lintjs: `npx eslint '**/*.js'`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the cse110-fa21-group2@1.0.0 lintjs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -11,5 +11,6 @@ module.exports = {
     ecmaVersion: 13,
   },
   rules: {
+    'linebreak-style': 0,
   },
 };

--- a/package.json
+++ b/package.json
@@ -5,10 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lintjs": "npx eslint '**/*.js'",
-    "lintcss": "npx stylelint '**/*.css'",
-    "lint": "npm run lintjs; npm run lintcss"
-
+    "lintjs": "npx eslint \"**/*.js\"",
+    "lintcss": "npx stylelint \"**/*.css\"",
+    "lint": "npm run lintjs && npm run lintcss"
   },
   "repository": {
     "type": "git",

